packages/monaco/src/browser/monaco-loader.ts

```diff
@@ -34,7 +34,7 @@ export function loadVsRequire(context: any): Promise<any> {
                     context.require = originalRequire;
                 }
                 resolve(amdRequire);
-            });
+            }, { once: true });
```

Comment:
Is it temporary, right? as well as changes to `monaco-electron-module`?

Reply:
I wish it was. The `once: true` is just to clean the listener on trigger because it has to be executed only once anyway (with the `resolve` call).

But the patch in `monaco-electron-module` is required for the monaco editor to load inside Electron when displaying http content.

The problem comes from here: https://github.com/Microsoft/vscode-loader/blob/master/src/core/scriptLoader.ts#L48

`vscode-loader` assumes a node environment when on Electron, and proceeds to look for the dependencies on the application's host disk... Which is not the right place to look when connected to an http server.

Reply:
Actually maybe by setting a `preload` script for the electron-window will work...
Still trying things.